.codegen/tests/cmd/workspace/test.toml.tmpl

```diff
@@ -0,0 +1,100 @@
+# Code generated from OpenAPI specs by Databricks SDK Generator. DO NOT EDIT.
+{{- $excludes :=
+	list
+		"command-execution"
+		"statement-execution"
+		"dbfs"
+		"dbsql-permissions"
+		"account-access-control-proxy"
+		"files"
+        "dashboards"
+        "queries-legacy"
+}}
+
+{{- $seen := list "" }}
+{{- $cmdGrp := .KebabName }}
+{{- if or .HasParent (or .IsAccounts (in $excludes .KebabName))  }}
+{{skipThisFile}}
+{{- end}}
+{{- $generated := false -}}
+{{- $supportedMethods := list "get" "list" "list-pipelines" }}
+{{- range .Methods}}
+{{- if in $supportedMethods .KebabName}}
+{{- $generated = true }}
+
+[[Server]]
```

Comment:
Instead of defining a server definition for every command, should we generate a FakeWorkspace, extending the idea in `libs/testserver/fake_workspace.go` to support stubs for all API calls?

The advantage of fake workspace is eventually it'll be possible to do more CRUD-like autogenerated tests as well. It's also a more general interface, which can be useful in other contexts since it's basically a local simulation of a Databricks workspace.

Writing the server in Go also allows us handle all sorts of API paths and support CREATE, DELETE etc since generating the server stubs in Go is a lot more powerful that generating them in `toml`.